controllers/greptimedbcluster/deployers/meta.go

```diff
@@ -360,3 +361,16 @@ func (b *metaBuilder) containerPorts() []corev1.ContainerPort {
 		},
 	}
 }
+
+func (b *metaBuilder) env() []corev1.EnvVar {
+	return []corev1.EnvVar{
+		{
+			Name: "POD_IP",
```

Comment:
suggestion: should we make `POD_IP` as a constant?

Reply:
souds reasonable to me.